packages/shared/src/components/buttons/ClickableText.tsx

```diff
@@ -8,6 +8,8 @@ export interface BaseClickableTextProps {
   pressed?: boolean;
   tag?: React.ElementType & AvailableTags;
   defaultTypo?: boolean;
+  flexRowApplied?: boolean;
+  underlined?: boolean;
```

Comment:
These properties will violate the guidelines we have for `ClickableText`. I think this shouldn't be necessary.